Userland/Libraries/LibCore/System.cpp

```diff
@@ -960,6 +960,14 @@ ErrorOr<void> socketpair(int domain, int type, int protocol, int sv[2])
     return {};
 }
 
+ErrorOr<Array<int, 2>> pipe()
```

Comment:
In order for commits to remain atomic, this commit should come before the commit that uses it